spec/system/project_spec.rb

```diff
@@ -396,7 +396,22 @@
         expect(page).to have_content("You have a background job running.")
         expect(sponsor_user.user_jobs).not_to be_empty
         user_job = sponsor_user.user_jobs.first
-        expect(user_job.job_id).to eq(job.job_id)
+        expect(user_job.job_id).not_to be nil
+      end
+
+      it "renders the completion time for the Sidekiq job" do
+        visit project_contents_path(project_not_in_mediaflux)
+        expect(page).to have_content("List All Files")
+        click_on "List All Files"
+        wait_for_ajax
+        # I cannot determine how to avoid this
+        sleep 1
```

Comment:
Prefer to check the page contents for the pop up contents, vs sleep.  I believe this will work, but not 100% sure
```suggestion
        expect(page).to have_content("... contents of pop up ...)
```

Reply:
I apologize, I have attempted this with https://github.com/pulibrary/tiger-data-app/pull/497/commits/966613bb99a090f88a1e67967035e5556daae4de, and this does not seem to be passing either locally, or on CircleCI. I am going to restore this, but please correct me if there was an error in my attempt.